layout/css/hurricane-svg.css

```diff
@@ -48,7 +48,7 @@ line, polyline, path{
 
 #map-elements-portrait, #map-elements-portrait-top, #map-elements-portrait-mid {
   /* doesn't work in IE */
-  transform: translate(-50px, -118px) scale(1.5);
+  transform: translate(-50px, -163px) scale(1.55)
```

Comment:
👍  There was some weird scaling on mobile when trying to fill the window with counties and prcp.